docs/editors-guide/rare-disease-subset.md

```diff
@@ -0,0 +1,49 @@
+# Rare Disease Subset in Mondo
+
+A rare disease is a disease that affects fewer than 200,000 people in the United States. However, this definition is not universal, and it varies between rare disease sources, geographical locations, etc. Consequently, rare disease criteria also vary between sources, and one might have difficulties reconciling sources and creating a list for rare diseases. Mondo recognizes these impediments and differences of view. This document explains how Mondo handles rare diseases, how one could get a rare disease list, and the efforts made to bring the community together.
+
+#### What are rare disease in Mondo?
+We rely on rare disease authorities (see table below) to determine all diseases recognized as “rare”, according to their own criteria. When a disease is considered “rare” according to a source, the corresponding Mondo term is annoted with `in_subset` annotation expressing that this disease is considered rare according to this source: `X_rare`  (where X is the rare disease authority).
+
+example: calciphylaxis (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
+![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)
+
+
+In our [2021 publication](https://mondo.monarchinitiative.org/pages/analysis/), we identified rare diseases as 
+- terms annotated with the `‘bearer of’ some rare` axiom (as provided by Mondo contributing resources)
```

Comment:
This is IMO too weak.. Is there any hope we can drop this phrase and replace it by the operational criteria that _led us_ to assert that axiom?

Reply:
We 'could' but these came before my time, and I don't know if there were any operational criteria... 

@nicolevasilevsky, do you know how these "bearer of rare" axiom were created? All I could find is that it was provided by the sources.

Reply:
Note to self: can also add this reference: https://pubmed.ncbi.nlm.nih.gov/32020066/